config/cookstyle.yml

```
@@ -1154,6 +1154,16 @@ Chef/Deprecations/FoodcriticTesting:
     - '**/Rakefile'
     - '**/Gemfile'
 
+Chef/Deprecations/DeprecatedYumRepositoryActions:
+  Description: With the release of Chef Infra Client 12.14 and the yum cookbook 3.0 several properties in the yum_repository resource the actions the yum_repository resource actions were renamed. The `add` action became `create` and `delete` became `remove` to better match other resources in Chef Infra Client.
```

```suggestion
  Description: With the release of Chef Infra Client 12.14 and the yum cookbook 3.0 several actions in the yum_repository resource actions were renamed. The `add` action became `create` and `delete` became `remove` to better match other resources in Chef Infra Client.
```

This sentence seemed mid-revision so I tried to guess the intent.